src/gui/main.cc.in

```diff
@@ -19,9 +19,9 @@ void setupApplication ()
   QCoreApplication::setApplicationName("gepetto-gui");
   QCoreApplication::setApplicationVersion("@PROJECT_VERSION@");
   
-  QStringList theme_paths;
-  theme_paths << "/usr/local/share/icons/";
-  QIcon::setThemeSearchPaths(theme_paths);
+  QStringList theme_paths = QIcon::themeSearchPaths();
+  if (!theme_paths.contains("/usr/local/share/icons/"))
+    theme_paths.prepend("/usr/local/share/icons/");
```

Comment:
You should add the missing `QIcon::setThemeSearchPaths(theme_paths);` to make things work.